pyaerocom/aeroval/coldatatojson_engine.py

```diff
@@ -225,6 +225,9 @@ def process_coldata(self, coldata: ColocatedData):
 
             _write_site_data(ts_objs, out_dirs["ts"])
 
+            scatter_freq = min(TsType(fq) for fq in self.cfg.time_cfg.freqs)
+            scatter_freq = str(min(scatter_freq, main_freq))
```

Comment:
please keep the `scatter_freq` type to `TsType` here and convert to string on the `_process_map_and_scat` call (if needed).

Reply:
👍